mor/mor/feel/feelmor/parameterspace.hpp

```diff
@@ -1463,6 +1476,35 @@ class ParameterSpace: public CommObject, public std::enable_shared_from_this<Par
             //M_max.setParameterSpace( this->shared_from_this() );
         }
 #endif
+    //! constructor from ModelProperties
+    ParameterSpace( ModelParameters const& modelParameters, worldcomm_ptr_t const& worldComm = Environment::worldCommPtr() )
+        :
+        super( worldComm ),
+        M_nDim(),
+        M_min(),
+        M_max(),
+        M_mubar()
```

Comment:
why mubar should be stored here ?
it should be associated to the associtated method

Reply:
still, wouldn't it be usefull to have a default value in the `parameterSapce`, independently of the method used ?

Reply:
what would be the purpose ? What would be the meaning ?

it has to be intrinsic to ParameterSpace

don't we have `center()` or logCenter()` ? can't it be that ? or `min()` or `max()` ? those are specific interesting values()

Reply:
ok, I understand. I will "undo" the corresponding commits for the PR, because some other works has also been made in this branch

Reply:
the use case I'm thinking of, is in the case of a geometric parameter. Mubar could represent the parameter used for the reference domain. It could be different of min, max, center or logCenter

Reply:
@prudhomm I only commented the corresponding lines in the code, it is the right thing to do, or should I delete all the  lines ?

Reply:
@romainhild it is specfic to the method that will handle the geometric mapping, I don't see why it should be in ParameterSpace.

Reply:
@thomas-saigre the code should be simply removed